### DISCUSSION TITLE
Uno.WinUI: MediaPlayerElement on Android

### DISCUSSION THREAD

**Discussion by gevart** [unoplatform/uno#11695] [Q&A] [answered]
I've just created a fresh Uno WinUI solution, updated Uno nugets to the latest dev, also updated WindowsAppSDK to 1.2.x as it was stated that MediaPlayerElement for WinUI requires version 1.2+. Added uses-permissions to manifest as written in the Uno MediaPlayerElement docs.
Still it works on Win head but shows nothing on Android Emulator (Android 7.1 if that matters).
I've attached the solution, maybe it can help identify the problem
[MediaPlayer.zip](https://github.com/unoplatform/uno/files/10975947/MediaPlayer.zip)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report! Can you try out again with the latest experimental builds?

**Reply by gevart** [reply]
Sorry for the delay, I've updated Uno nugets to the latest dev (from 49 to 109) and now it's working fine. Thanks!
